src/Illuminate/Mail/Events/MessageSent.php

```diff
@@ -46,14 +46,10 @@ public function __serialize()
     {
         $hasAttachments = collect($this->message->getAttachments())->isNotEmpty();
 
-        return $hasAttachments ? [
-            'sent' => base64_encode(serialize($this->sent)),
-            'data' => base64_encode(serialize($this->data)),
-            'hasAttachments' => true,
-        ] : [
+        return [
             'sent' => $this->sent,
```

Comment:
This never needs to be base 64 encoded and serialized anymore?

Reply:
Admittedly, I don't know all of the use cases here, but my thinking is: if `SentMessage` is being base64_encoded for attachments when it is serialized, then there's no need to encode it again when `MessageSent` is being serialized.

Do you see a flaw in that thinking?

Reply:
Yeah, that could be true.